.ci/lib.sh

```diff
@@ -15,6 +15,9 @@ export KATA_RUNTIME=${KATA_RUNTIME:-kata-runtime}
 # more formats).
 export KATA_DOCKER_TIMEOUT=30
 
+# Ensure GOPATH set
+export GOPATH=${GOPATH:-$(go env GOPATH)}
```

Comment:
I wonder if we ever run this script before a system has go installed?
just a thought. We could maybe caveat it with something like (pseudo code)

```
if [ command go ]; then
 export GOPATH=${GOPATH:-$(go env GOPATH)}
fi
export GOPATH=${GOPATH:-${HOME}/go}
```

just a thought.